lib/services/ipmi-obm-service.js

```diff
@@ -61,6 +61,12 @@ function ipmiObmServiceFactory(BaseObmService, Promise) {
         return this._runInternal(['chassis', 'bootdev', 'pxe']);
     };
 
+    IpmiObmService.prototype.forceBootPxe = function() {
```

Comment:
@pengz1  I'm not sure what's the better function name here, but could you help add some annotations about the difference with 'setBootPxe' so that the developers/users wouldn't be confused when reviewing this parts of codes in future